modules/installation-osp-enabling-swift.adoc

```diff
@@ -35,3 +35,8 @@ $ openstack role add --user <user> --project <project> swiftoperator
 ----
 
 Your {rh-openstack} deployment can now use Swift for the image registry.
+
+[NOTE]
+====
+{rh-openstack} with the Ceph storage back end only supports the `.raw` image format. You must store the CoreOS image in the `.raw` format. Follow link:https://access.redhat.com/documentation/en-us/red_hat_openstack_platform/16.2/html/creating_and_managing_images/ch-image-service#section-convert-image[the manual process] to transform the image from `.qcow2` to `.raw`.
```

Comment:
The note's wording is a bit unclear to me, so I'd suggest clarifying whether you mean that RHOSP with Ceph supports only `.raw` or if only _RHOSP with Ceph back end_ is the only way to support `.raw`. I think it's the former? Anyway, something like this (if this wording is correct): 
```suggestion
{rh-openstack} with the Ceph storage back end supports only the `.raw` image format. You must store the {op-system} image in the `.raw` format. Follow link:https://access.redhat.com/documentation/en-us/red_hat_openstack_platform/16.2/html/creating_and_managing_images/ch-image-service#section-convert-image[the manual process] to transform the image from the `.qcow2` to `.raw` format.
```

Reply:
Addressed